### DISCUSSION TITLE
Have eternal slayer ring contribute to bank value just like graceful

### DISCUSSION THREAD

**Discussion by snappie** [runelite/runelite#15091] [Q&A] [answered]
Another user already asking this in issue #8508.
However the answer given is inconsistent with other items like the graceful set, which do contribute to bank value
It seems reasonable to overturn that decision

**Comment by Bram91** [comment, marked as answer]
Graceful items can be sold back for marks of grace and you can buy amylase with the returned marks, so graceful does have a value. The eternal slayer ring has no value because there is no way to get the gem back.
